radicle/src/git.rs

```diff
@@ -16,7 +16,7 @@ pub use ext::Error;
 pub use ext::Oid;
 pub use git2 as raw;
 pub use git_ref_format as fmt;
-pub use git_ref_format::{refname, Component, Qualified, RefStr, RefString};
+pub use git_ref_format::{lit, name, refname, Component, Namespaced, Qualified, RefStr, RefString};
```

Comment:
This diff line reminds me of when Kim would patch my Haskell code but first he would import half of Control.Applicative

Reply:
lol, that's how you know it's a good abstraction -- everything is immensely useful ;P

I'm not the biggest fan of this re-exporting style. The rust-analyzer shortcut for importing will always import from the crate itself, and I never knew what was re-exported from here or not.

Reply:
Yeah I mean I have it imported as `fmt` above, so you could also do `fmt::lit`, `fmt::name` etc.

Reply:
Right, is that the standard to go by then or could I convince you to just import the crate where needed :eyes:? Would you want me to fix that up before merging?